portfolio/src/main/webapp/aboutme.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>About Me</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+    <style>
+        li a {
+        display: block;
+        color: #343;
+        padding: 8px 16px;
+        text-decoration: none;
+        }
+
+      li a:hover {
+        background-color: yellow;
+        color: purple;
+       }
+
+      ul {
+        padding: 0;
+        margin: 0;
+        width: 200px;
+        background-color:grey;
+         }
+    </style>
+  </head>
+  <body>
+    <div id="content">
+      <h1> Welcome!</h1>
+      <p style="font-family:Helvetica;">PS4, my dog, anime, basketball, netflix, CS. I'm a pretty simple dude. I go to Howard (shoutout my Drew Kings!)</p>
```

Comment:
Please use css class selectors to apply style to tags instead of inline!

Reply:
Yeah this was a definitely a oof moment for me because I knew having style tags in each one of these html folders when they were all the same thing didn't make sense to me.